Sources/StreamChatSwiftUI/ChatChannel/Composer/MessageComposerViewModel.swift

```diff
@@ -222,6 +259,17 @@ open class MessageComposerViewModel: ObservableObject {
                 _ = url.startAccessingSecurityScopedResource()
                 return try AnyAttachmentPayload(localFileURL: url, attachmentType: .file)
             }
+            attachments += try addedVoiceRecordings.map { recording in
+                _ = recording.url.startAccessingSecurityScopedResource()
```

Comment:
Is `recording.url` a bookmark? If not, `startAccessingSecurityScopedResource` probably isn't necessary.

Reply:
We had issues with regular file attachments because we didn't have this. I would prefer to keep it, unless you see some risks?